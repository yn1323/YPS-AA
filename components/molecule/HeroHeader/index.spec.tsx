import * as React from 'react'
import renderer from 'react-test-renderer'
import Component from '.'
import * as nextImage from 'next/image'

Object.defineProperty(nextImage, 'default', {
  configurable: true,
  value: (props: any) => <img {...props} />,
})

test(Component.name, async () => {
  const component = await renderer.create(<Component />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
