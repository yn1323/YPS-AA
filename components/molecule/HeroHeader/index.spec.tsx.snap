// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroHeader 1`] = `
<div
  className="MuiBox-root MuiBox-root-1 HeroHeader__BackGround-sc-12aeg85-3 kIFFBp"
>
  <img
    alt="Picture of the author"
    height={250}
    src="/images/storyset/time.svg"
    width={250}
  />
  <div>
    <h6
      className="MuiTypography-root HeroHeader__CatchCopy-sc-12aeg85-1 kvNxwU MuiTypography-subtitle1"
    >
      共有できる無料のシフト管理
    </h6>
    <div
      className="HeroHeader__LoginButtonWrapper-sc-12aeg85-2 laPosn"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          ログインしてはじめる
        </span>
      </button>
    </div>
  </div>
</div>
`;
